CWE/localisation/Essential Localisation 2.csv

```diff
@@ -2965,4 +2965,14 @@ ina_casa_desc;The INA-Casa Plan, also known as Fanfani Plan (from the name of Am
 ITA_agrarian_reform_1950_title;Enact Legge Stralcio n.841;;;;;;;;;;;;;x;;;;
 ITA_agrarian_reform_1950_desc;The Legge Stralcio n.841 is a land reform law that seeks to emancipate the italian peasantry through the means of appropriation and redistribution, turning farmers into small business owners and destroying latifundism. The redistribution of the farmed land will also shift crop-production from an extensive to an intensive mode of production hence maximizing profits and efficiency.;;;;;;;;;;;;;x;;;;
 casmez_title;Fund for the South;;;;;;;;;;;;;x;;;;
-casmez_desc;The Fund for the South (Cassa per il Mezzogiorno), whose full name is Fund for extraordinary public works in the South of Italy (Cassa per opere straordinarie di pubblico interesse nell'Italia Meridionale), commonly shortened as Casmez, is a state-owned agency for the development and stimulation of the economy of the South of Italy, that has been lagging behind the North since the unification. The project was pushed forward by a group of economists basing themselves on Roosevelt's New Deal who proposed the government an action plan for the region. 7 Regions will be designated as beneficiaries of the plan, that will build industrial consortiums in key zones: Abruzzo e Molise, Calabria, Puglia, Campania, Basilicata, Sicily and Sardinia and the 2 Latium provinces of Latina and Frosinone.;;;;;;;;;;;;;x;;;;
\ No newline at end of file
+casmez_desc;The Fund for the South (Cassa per il Mezzogiorno), whose full name is Fund for extraordinary public works in the South of Italy (Cassa per opere straordinarie di pubblico interesse nell'Italia Meridionale), commonly shortened as Casmez, is a state-owned agency for the development and stimulation of the economy of the South of Italy, that has been lagging behind the North since the unification. The project was pushed forward by a group of economists basing themselves on Roosevelt's New Deal who proposed the government an action plan for the region. 7 Regions will be designated as beneficiaries of the plan, that will build industrial consortiums in key zones: Abruzzo e Molise, Calabria, Puglia, Campania, Basilicata, Sicily and Sardinia and the 2 Latium provinces of Latina and Frosinone.;;;;;;;;;;;;;x;;;;
+alliance_MedPac;Mediterranean Pact Member;;;;;;;;;;X;;;;;;;
+leave_alliance_MedPac;Left the Mediterranean Pact;;;;;;;;;;X;;;;;;;
```

Comment:
These 2 localisation entries use a capital 'X' instead of the small 'x' in the line ending. Maybe this is causing the error?